evm_loader/lib/src/account_storage.rs

```diff
@@ -562,15 +554,20 @@ impl<'a, T: Rpc> EmulatorAccountStorage<'_, T> {
                     unimplemented!();
                 }
             }
+        } else {
+            let account_data = AccountData::new_from_account(pubkey, &account);
+            self.mark_account(pubkey, false);
+            Ok(self
+                .accounts
+                .insert(pubkey, Box::new(RefCell::new(account_data))))
         }
     }
 
-    fn add_empty_account(&self, pubkey: Pubkey) -> NeonResult<&RefCell<AccountData>> {
+    fn add_empty_account(&self, pubkey: Pubkey) -> &std::cell::RefCell<AccountData> {
```

Comment:
`std::cell::RefCell` can also be just replaced with `RefCell`

Reply:
done